lang/Cargo.toml

```diff
@@ -58,3 +58,4 @@ borsh = "0.10.3"
 bytemuck = "1"
 solana-program = "2"
 thiserror = "1"
+shrinkwraprs = "0.3.0"
```

Comment:
It would be better if we could avoid adding this dependency because adding dependencies for simple tasks increases the attack surface unnecessarily.

```suggestion
```